package exception_handing;

public class Throws_Test_6 {

	public static void main(String[] args) {
		//used in method definition ,
		//to declare that a risky method is being called.

	}

}
